server/docker-compose.yml

```diff
@@ -1,22 +0,0 @@
-version: "2"
```

Comment:
any reason to delete this?
want to make sure team members' toes aren't being stepped on